.rubocop.yml

```diff
@@ -133,3 +133,7 @@ Style/HashTransformValues:
 Rake/Desc:
   Exclude:
     - 'lib/tasks/reset_qa.rake'
+
+RSpec/NestedGroups:
+  Exclude:
+    - 'spec/models/user_spec.rb'
```

Comment:
The default nesting is 3 and I've gone to 5. I have change the global level to 5 to get around it. I feel the extra nesting here makes it more readable. If I leave it at 3 and change the tests I feel it's much harder to comprehend.

On a different note, these tests are doing more than they probably should. They probably belong more in the spec for `UserChangeForVersion` / `Computacenter::UserChangeGenerator` which doesn't exist. However I wanted to test through the `User` model to ensure `User`, `paper_trail` and `Computacenter::UserChangeGenerator` all worked in unison.

Reply:
ah, you changed the rules codebase-wide! :) I'm going to ping-in @tonyheadford for a second opinion here, so that it can be a team decision. 

I agree that the spec here is doing probably too much for a single model spec - maybe the working-in-unison tests could/should be an `integration` spec instead?

Reply:
I've been caught by this one and think that it should be increased above 3, in my instance I think it made the specs more difficult to read to flatten into 3 levels. Not sure what the limit should be but agree it should be higher

Reply:
5 sounds perfect then 😉

Reply:
Alright, I'm happy with that - ship it! 👍   🚀